src/ol-layerswitcher.js

```diff
@@ -183,9 +183,33 @@ export default class LayerSwitcher extends Control {
         if (lyr.getLayers && !lyr.get('combine')) {
 
             li.className = 'group';
+            
+            //Group folding
+            var ulStyle, ulHeight;
+            if (lyr.get('unfolded')) {
+              lyr.id = lyrId;
+              var fold = document.createElement('i');
```

Comment:
If we use CSS for the arrow then we might be able to remove the need for an extra element and instead style the group title and make the group title clickable?